server/config/routes/api/v21.rb

```diff
@@ -9,6 +9,7 @@
             get :validate
             get :validate_cdn
             get :log
+            get :get_openshift_disk_space
```

Comment:
I know I was forced to make some of these verbs, but for my sanity, how about we remove the get since this is already provided by the HTTP verb and fits in pretty easily.

```ruby
get 'openshift_disk_space', to: 'deployment#get_openshift_disk_space'
```




http://goo.gl/zRFMrj
http://goo.gl/exeYRP